composables/useActiveGrainType.ts

```diff
@@ -1,64 +1,17 @@
-// const globalState = reactive({
-//   someString: "Initial value",
-//   someBoolean: false
-// });
+// can also use: "12 кукур-Table 1"
+const DEFAULT_GRAIN_TYPE = "10 пшенЯР-Table 1";
```

Comment:
This shouldn't be here, it should be set from scrollytelling and accepted as an argument

Reply:
my thinking was the scrollytelling would change it using `setActiveGrainType` which is available from the composable

Reply:
cool I didn't see that 👍